packages/renderer/src/lib/configmaps-secrets/ConfigMapSecretEmptyScreen.svelte

```diff
@@ -1,10 +1,32 @@
 <script lang="ts">
+import { onMount } from 'svelte';
+
+import { getResourcePermitted } from '/@/stores/kubernetes-context-permission';
+import { kubernetesContexts } from '/@/stores/kubernetes-contexts';
+
 import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
 import KubernetesEmptyScreen from '../kube/KubernetesEmptyScreen.svelte';
+
+let contextName = $derived($kubernetesContexts.find(c => c.currentContext)?.name);
+let experimentalStates: boolean = $state(false);
+let configMapsSecretsPermitted: boolean = $derived(
```

Comment:
Configmaps could be accessible and secrets not or vice-versa, but message only changes when neither is? Ditto for ingress/routes.